payjoin/src/receive/v2/mod.rs

```diff
@@ -908,6 +901,21 @@ fn subdir(directory: &Url, id: &ShortId) -> Url {
     url
 }
 
+/// Creates a Payjoin URI from a session context
+pub fn create_pj_uri<'a>(
```

Comment:
Nit: I would remove the verb here and just cal lit `pj_uri` to be more consistent with the rest of the codebase